include/irr/asset/IAsset.h

```diff
@@ -191,6 +191,11 @@ class IAsset : virtual public core::IReferenceCounted
 		*/
 		virtual void convertToDummyObject(uint32_t referenceLevelsBelowToConvert=0u) = 0;
 
+        void convertToDummyObject_common(uint32_t referenceLevelsBelowToConvert)
```

Comment:
inline

Reply:
also what's the convention on metadata upon conversion to dummy object?

Reply:
nothing happens with metadata

Reply:
hmm ok, we might want to revist that in the far future.